src/components/navigation/z-navigation-tabs/index.tsx

```diff
@@ -268,7 +271,13 @@ export class ZNavigationTabs {
       tab.setAttribute("role", "tab");
       tab.tabIndex = -1;
     });
-    this.tabs[0].tabIndex = 0;
+
+    // pre-selected tab (a11y purpose)
+    const preselectedTab = this.tabs.findIndex((tab) => tab.ariaSelected === "true");
+    const activeTab = preselectedTab > 0 ? preselectedTab : 0;
```

Comment:
La navigazione viene usata anche senza avere una tab già selezionata al caricamento della pagina, quindi forse sarebbe meglio lasciare anche questa casistica.

Reply:
se non trova tab  preselezionata ( quindi `preselectedTab = -1`) l'active tab è la prima.

Reply:
Esatto, il problema è proprio quello: manca il caso in cui non c'è una tab selezionata.